app/src/main/res/values/strings.xml

```diff
@@ -116,4 +116,7 @@
     <string name="apps">Apps</string>
     <string name="are_you_sure">Are you sure?</string>
     <string name="view">View</string>
+    <string name="delete_warning">Are you sure you want to delete it? You will not be able to recover after deletion.</string>
```

Comment:
Maybe it will be better to replace `You will not be able to recover after deletion.` with `This action cannot be undone.`?